torchvision/io/__init__.py

```diff
@@ -1,9 +1,3 @@
-from typing import Any, Dict, Iterator
-
-import torch
-
-from ..utils import _log_api_usage_once
```

Comment:
These are no longer needed. `flake8` didn't pick up on this since we blanket ignore `F401`, i.e. unused imports, in `__init__.py` files:

https://github.com/pytorch/vision/blob/c4c0ef98d06c7011d80991827b099a6266b7075c/setup.cfg#L14-L15